backend/src/db/migrations/20230826133720_add_role_id_to_users_table.ts

```diff
@@ -0,0 +1,29 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'users';
+
+const REFERENCE_COLUMN = 'role_id';
+
+const ROLE_MAPPING = {
+  CHATBOT: 1,
+  USER: 2,
+};
```

Comment:
we need to find role by key, not by id
```suggestion
const USER_ROLE_KEY = 'user'
```

Reply:
Changed it!